addons/web/static/src/js/core/utils.js

```diff
@@ -756,6 +829,29 @@ var utils = {
             }
         });
     },
+    /**
+     * Serialize and stringify a given object. Multiple consecutive backslashes
+     * are then compressed to avoid having an exponential amount of them.
+     *
+     * @param {any} object
+     * @returns {string}
+     */
+    stringify: function (object) {
+        const stringified = JSON.stringify(object, _jsonReplacer);
+        const compressed = stringified.replace(/\\{2,}/g, compressBS);
+        return compressed;
+    },
+    /**
+     * Parse and instantiate strings serialized via 'stringify'.
+     *
+     * @param {string} string
+     * @returns {any}
+     */
+    parse: function (string) {
+        const decompressed = string.replace(/\\\d+/g, decompressBS);
```

Comment:
it could conflict... JSON.parse("'coucou\\3JUM'") I guess...

Reply:
Done